racket/collects/raco/raco.rkt

```diff
@@ -38,13 +39,48 @@
   ((executable-yield-handler) result) ; to enable GUI commands
   (exit result))
 
+(define (parse-cmd-output-option cmdline)
+  (let* ([tout-idx (index-of cmdline "--cmd-output")]
+         [cmd-is-not-help (not (and (pair? cmdline)
+                                    (equal? "help" (car cmdline))))])
```

Comment:
I thought that at this point, `raco help` is converted to `raco ... --help` already, no?